.env

```diff
@@ -0,0 +1 @@
+ENABLE_NEW_NAVIGATION=yes
```

Comment:
No, because `dotenv` is only included in development and test in the Gemfile. This file will be ignored in all other environments.